src/generated/Normalizer/PriceNormalizer.php

```diff
@@ -75,6 +85,16 @@ public function normalize($object, $format = null, array $context = array())
         else {
             $data->{'value'} = null;
         }
+        if (null !== $object->getAppliedPromotions()) {
+            $values = array();
```

Comment:
```suggestion
            $values = [];
```

Reply:
This file is automatically generated by jane, I would not change it

Reply:
Oh right